concrete_biopython/FheSeq.py

```diff
@@ -104,6 +110,7 @@ def __eq__(self, other):
         if len(self) != other.size:
             return fhe.zero() # return False if the lengths are different
         else:
+            #return np.sum(self._data == other) == len(self)
             return (len(self) - np.sum(self._data == other))==0
```

Comment:
This will require changes for the next version of Concrete. Comparisons will become multi precision and that sum would be possible to overflow without a specific input in the inputset (e.g., if the maximum value of sum is 10 in the inputset, but could be 20 in runtime). To overcome the issue, you'd use:
```python
fhe.hint(np.sum(self._data == other), can_store=len(self))
```
which would tell the compiler to assign a bit width that can store the specified value. In this case sum can go up to `len(self)` so we hint with that.